src/components/shared/PasswordInput/index.tsx

```diff
@@ -1,47 +1,45 @@
-import React, { useState } from 'react';
+import React, { useState, forwardRef } from 'react';
 import { IconButton, InputAdornment } from '@mui/material';
 import { StyledInputProps } from 'src/components/shared/StyledInput/types';
 import CustomVisibleIcon from 'src/assets/icons/eye.svg';
 import CustomHiddenIcon from 'src/assets/icons/eye-closed.svg';
 import StyledPasswordInput from './styledPasswordInput';
 
-function PasswordInput({
-  placeholder,
-  width,
-  padding,
-  name,
-  stylevariant,
-}: StyledInputProps): JSX.Element {
-  const [showPassword, setShowPassword] = useState(false);
-  const handleClickShowPassword = () => setShowPassword(!showPassword);
-  const handleMouseDownPassword = (
-    event: React.MouseEvent<HTMLButtonElement>
-  ) => {
-    event.preventDefault();
-  };
+const PasswordInput = forwardRef<HTMLInputElement, StyledInputProps>(
+  ({ placeholder, width, padding, name, stylevariant, ...props }, ref) => {
+    const [showPassword, setShowPassword] = useState(false);
+    const handleClickShowPassword = () => setShowPassword(!showPassword);
```

Comment:
empty line above, I assume team should add some rule to eslint to separate code, we can have everything without separation its difficult to read

Reply:
add new rule to eslint to have empty lines